packages/ui-components/src/components/HeaderContainer/HeaderContainer.component.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Juno contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from "react"
+
+const headerContainerStyles = `
+  jn-flex
+  jn-flex-col
+  jn-sticky
+  jn-top-0
+  jn-z-50
+  jn-shadow-lg
+  jn-bg-theme-global-bg
+`
+
+export const HeaderContainer = ({
+  fullWidth = false,
+  className = "",
+  children = null,
+  ...props
+}: HeaderContainerProps) => {
+  return (
+    <div
+      className={`
```

Comment:
**(Non-Blocking Improvement)**

In other components, we try extract the CSS classes into a variable so the component is easier to read.
Could we extract into `const containerStyles`, or something like that?

Reply:
I'm not quite sure about this point. Here there are already `const headerContainerStyles` with the main styles..